azavea/raster-foundry
# Fix disabling color correction groups

## Overview

Fix disabling color correction groups

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~

### Demo

![image](https://cloud.githubusercontent.com/assets/4392704/22519295/d156e83c-e87d-11e6-93fd-3bfe2d18be26.png)


### Notes

Contrast breaks stuff when set to null, so it is set to 0 instead. 
I also changed the min/max slider to allow the full 16-bit color range.
When a project is first created, it will by default have color correction disabled.  The mosaic doesn't have a color correction profile defined, which seems to make the scene not show up. 

## Testing Instructions

 * Color correct an ingested scene
 * Verify that disabling color correction groups does not result in bad behavior (scene not showing up, etc)
* Verify that parameters are being set to null in chrome dev network tab (except for contrast, which should be 0)
* Verify that disabling a color correction group is correctly persisted across reloads
